SpatialGDK/Source/SpatialGDK/Private/Schema/CrossServerEndpoint.cpp

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Schema/CrossServerEndpoint.h"
+
+namespace SpatialGDK
+{
+CrossServerEndpoint::CrossServerEndpoint(Schema_ComponentData* Data)
+	: ReliableRPCBuffer(ERPCType::CrossServer)
+{
+	ReadFromSchema(Schema_GetComponentDataFields(Data));
+}
+
+void CrossServerEndpoint::ApplyComponentUpdate(Schema_ComponentUpdate* Update)
```

Comment:
would appreciate comments throughout this file describing what this is doing- or at least a pointer to where it is described. For example, it's not immediately clear why the clear field logic in this function is the way that it is.

Reply:
Right, that's me enforcing that both fields (which should be valid/invalid together) are cleared at the same time. But I'm just overzealous.